app/controllers/dashboard_controller.rb

```diff
@@ -3,6 +3,7 @@
 # DashboardController shows the main page.
 class DashboardController < ApplicationController
   include Discovery
+  before_action :update_grains
```

Comment:
Do we really need to do this? It will impact all requests done to this controller, that now will have to wait for the request to the salt-api.

Reply:
probably not, that's why i have added it as a separate commit. wanted to get some feedback. yes the idea was to always show the latest grains, hence putting it in the application controller. but if that's too invasive, lets drop it again